### PR TITLE
 syslog-ng: add Premium Edition 6 and 7 support

### PR DESCRIPTION
**Pull request by oalbrigt**

### CODE REVIEW

**Comment by jszigetvari** [comment]
The `#!@BASH_SHELL@` also gave me headaches, but I assume this part is due to be substituted by a make script or something similar

File: heartbeat/syslog-ng.in [hunk 1]
```diff
@@ -168,7 +223,17 @@ start_syslog_ng()
 
 stop_syslog_ng()
 {
-	pkill -TERM -f "$PROCESS_PATTERN"
+	if [ -x "$SYSLOG_NG_CTL" ]; then
+		if [ -n "${OCF_RESKEY_control_file}" ] && [ -S "${OCF_RESKEY_control_file}" ]; then
+			"$SYSLOG_NG_CTL" stop --control "$CONTROL_FILE"
```
[the remark this answers is not in the document]
Comment (oalbrigt) [reply heartbeat/syslog-ng.in]: Oh. That makes sense.

**Comment by oalbrigt** [comment]
Yeah, that's because of new detection of where bash is, so if you run "./autogen.sh && ./configure" it will be generated without .in with the correct bash-path.

**Comment by jszigetvari** [comment]
@oalbrigt Apart from the above comments, the RA seems to work well enough for me.

**Comment by oalbrigt** [comment]
@jszigetvari Great. Thanks for your initial code, and additional testing.

**Pull request merged**